### PR TITLE
mkfs.xfs: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Managor** [comment]
Page style copied from mkfs.ext4.md

File: pages/linux/mkfs.xfs.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+> Create an xfs filesystem inside a partition.
+> More information: <https://manned.org/mkfs.xfs>.
+
+- Create an xfs filesystem inside partition 1 on device b (`sdb1`):
+
+`sudo mkfs.xfs {{/dev/sdb1}}`
+
+- Create an xfs filesystem with a volume-label:
+
+`sudo mkfs.xfs -L {{volume_label}} {{/dev/sdb1}}`
```
Comment (spageektti) [inline pages/linux/mkfs.xfs.md:12]:
```suggestion
`sudo mkfs.xfs -L {{volume_label}} {{/dev/sdX1}}`
```
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases

File: pages/linux/mkfs.xfs.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# mkfs.xfs
+
+> Create an xfs filesystem inside a partition.
+> More information: <https://manned.org/mkfs.xfs>.
+
+- Create an xfs filesystem inside partition 1 on device b (`sdb1`):
```
Comment (spageektti) [inline pages/linux/mkfs.xfs.md:6]:
```suggestion
- Create an xfs filesystem inside partition 1 on a device (`X`):
```

File: pages/linux/mkfs.xfs.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+`sudo mkfs.xfs {{/dev/sdb1}}`
```
Comment (spageektti) [inline pages/linux/mkfs.xfs.md:8]:
```suggestion
`sudo mkfs.xfs {{/dev/sdX1}}`
```

**Comment by Managor** [comment]
Someone should go through all the mkfs pages and fix them like this.

File: pages/linux/mkfs.xfs.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+> Create an xfs filesystem inside a partition.
+> More information: <https://manned.org/mkfs.xfs>.
+
+- Create an xfs filesystem inside partition 1 on a device (`X`):
+
+`sudo mkfs.xfs {{/dev/sdX1}}`
+
+- Create an xfs filesystem with a volume-label:
+
+`sudo mkfs.xfs -L {{volume_label}} {{/dev/sdX1}}`
```
Comment (acuteenvy) [inline pages/linux/mkfs.xfs.md:12]: XFS is written in capital letters in the man page.

```suggestion
> Create an XFS filesystem inside a partition.
> More information: <https://manned.org/mkfs.xfs>.

- Create an XFS filesystem inside partition 1 on a device (`X`):

`sudo mkfs.xfs {{/dev/sdX1}}`

- Create an XFS filesystem with a volume label:

`sudo mkfs.xfs -L {{volume_label}} {{/dev/sdX1}}`
```

**Review by acuteenvy — APPROVED** [review]

**Comment by spageektti** [comment]
> Someone should go through all the mkfs pages and fix them like this.

I will open a PR later.

**Review by spageektti — APPROVED** [review]

**Pull request merged**